pincer/objects/guild/guild.py

```diff
@@ -474,19 +477,326 @@ async def modify_member(
+        Returns
+        -------
+        :class:`~pincer.objects.guild.channel.Channel`
+            The new channel object.
+        """
+        ...
+
+    async def create_channel(
+        self,
+        reason: Optional[str] = None,
+        **kwargs
+    ):
+        data = await self._http.post(
+            f"guilds/{self.id}/channels",
+            data=kwargs,
+            headers=remove_none({"X-Audit-Log-Reason":reason))
+        )
+        return Channel.from_dict(construct_client_dict(self._client, data))
+
+    async def modify_channel_positions(
+        self, 
+        reason: Optional[str] = None,
+        *channel: Dict[str, Optional[Union[int, bool, Snowflake]]]
+    ):
+        """|coro|
+        Create a new channel object for the guild.
+
+        Parameters
+        ----------
+        reason : Optional[:class:`str`]
+            audit log reason |default| :data:`None`
+        \\*channel : Dict[str, Optional[Union[int, bool, :class:`~pincer.utils.snowflake.Snowflake`]
+            Keys:
+                - id : :class:`~pincer.utils.snowflake.Snowflake`
+                - position : Optional[:class:`int`]
+                - lock_permissions : Optional[:class:`bool`]
+                - parent_id : Optional[:class:`~pincer.utils.snowflake.Snowflake`]
+        """
+        await self._http.patch(
+            f"guilds/{self.id}/channels",
+            data=channel,
+            headers=remove_none({"X-Audit-Log-Reason":reason))
+        )
+
+    async def list_active_threads(self) -> Tuple[
+        Generator[Union[PublicThread, PrivateThread]], Generator[GuildMember]]:
+        """|coro|
+        Returns all active threads in the guild, including public and private threads.
+        
+        Returns
+        -------
+        Generator[Union[:class:`~pincer.objects.guild.channel.PublicThread`, :class:`~pincer.objects.guild.channel.PrivateThread`]], Generator[:class:`~pincer.objects.guild.member.GuildMember`]]
+            The new member object.
+        """
+        data = await self._http.get(f"guilds/{self.id}/threads/active")
+
+        threads = (
+            Channel.from_dict(construct_client_dict(self._client, data))
+            for channel in data["threads"]
+        )
+        members = (
+            GuildMember.from_dict(construct_client_dict(self._client, data))
+            for member in data["members"]
+        )
+
+        return threads, members
+
+    async def list_guild_members(self, limit: int = 1, after: int = 0):
+        """|coro|
+        Returns a list of guild member objects that are members of the guild.
+
+        Parameters
+        ----------
+        limit : int
+            max number of members to return (1-1000) |default| :data:`1`
+        after : int
+            the highest user id in the previous page |default| :data:`0`
+        """
+
+        members = await self._http.get(
+            f"guilds/{self.id}/members?{limit=!s}&{after=!s}"
```

Comment:
No need for `!s` since the variables are `int`s
```suggestion
            f"guilds/{self.id}/members?{limit=}&{after=}"
```

Reply:
bruh. I changed this because you told me to